satpy/readers/fci_l2_nc.py

```diff
@@ -32,8 +32,82 @@
 
 PRODUCT_DATA_DURATION_MINUTES = 20
 
+SSP_DEFAULT = 0.0
 
-class FciL2NCFileHandler(BaseFileHandler):
+
+class FciL2BaseClass(object):
```

Comment:
you don't need the 'Class' suffix

Reply:
Changed to FciL2CommonFunctions - sounds better i think